### ISSUE TITLE
The webapp docker-compose service does not resume automatically following reboot

### ISSUE THREAD

**Issue by tmcphillips** [openskope/skope-deployment#58]
I've noticed on all three of our servers (staging, app, and tacc-staging) that if the system reboots while the SKOPE services and applications are running, all of the docker containers resume running automatically following the boot--except for the webapp service (Meteor app) which exits with code 255, and the web (nginx) service which repeatedly starts, stops, and restarts repeatedly.

Manually issuing `docker-compose down` followed by `docker-compose up -d` cleans everything up quickly.  However, we need to make all services resume reliably on boot.

**Comment by jterstriep** [comment]
I've definitely seen everything come up fine in the past but I've also seen the webapp have problems restarting. Nginx is restarting because the webapp is running. If we fix the webapp then the nginx server should be fine.

**Comment by tmcphillips** [comment]
I was able to fix this problem by changing the dependencies between the _web_ (nginx) and _webapp_ (meteor) services in the docker-compose file.

Specifically, I edited docker-compose-staging.yml on tacc-staging to (1) remove the dependency of the _web_ (nginx) service on the _webapp_ service (and all other services), (2) make the _webapp_ (meteor) service wait for the _web_ service to start, and (3) make the _webapp_ service automatically restart if it stops.

Now when the SKOPE containers are started with docker-compose and the system is rebooted, all of the services resume cleanly on system restart.
